CourseSketchProjects/CourseSketchDatabaseServer/src/main/java/database/institution/mongo/AssignmentManager.java

```diff
@@ -89,7 +89,7 @@ private AssignmentManager() {
      */
     public static String mongoInsertAssignment(final Authenticator authenticator, final DB dbs, final String authId, final SrlAssignment assignment)
             throws AuthenticationException, DatabaseAccessException {
-        final DBCollection assignmentCollection = dbs.getCollection(ASSIGNMENT_COLLECTION);
+        final DBCollection assignmentCollection = dbs.getCollection(getCollectionFromType(School.ItemType.ASSIGNMENT));
```

Comment:
This variable has 3 names depending on which method it is used in: collection, assignments, and assignmentCollection. Can you just pick one of the 3 and standardize it?